integrations/quarkus/pom.xml

```diff
@@ -0,0 +1,101 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- when using this, the extension itself (deployment part) does not work anymore (The extension processor is not called anymore, BuildItems are not available and executed)
+      Due to differences in compiler ?? -->
+    <!--parent>
+        <groupId>one.microstream</groupId>
+        <artifactId>microstream-integrations-parent</artifactId>
+        <version>08.00.00-MS-GA-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent-->
```

Comment:
Commented code?

Reply:
When using the Parent, The Quarkus extension is not functional and tests and examples of using MicroStream with Quarkus fails.

@zdenek-jonas  If the parent is required for the release process, we either need to put this in a separate repository or we can't release this Quarkus extension at all.

Reply:
I have another idea to solve this problem, let's discuss it personally with the whole team.

Reply:
Integrated https://github.com/microstream-one/microstream/pull/423 in this PR